azext_iot/_help.py

```diff
@@ -1041,6 +1041,19 @@
         az iot edge set-modules --hub-name {iothub_name} --device-id {device_id} --content ../modules_content.json
 """
 
+helps[
+    "iot edge export-modules"
+] = """
+    type: command
+    short-summary: Export the edge module configuration of a single edge device.
+    long-summary: |
```

Comment:
In this long-summary it is unnecessary to use the `|` , you'd want to use it if there are newlines you want to render in a block of yaml.

Reply:
got it, thanks!